lib/core/host.js

```diff
@@ -17,6 +17,7 @@ export class Host<Detect, Opt: { [string]: ModuleOption<any> }> {
 	options: Opt | void;
 
 	detect: (a: URL, context: ?Thing) => void | null | false | Detect;
+	// handleLink: (href: string, detectResult: Detect | null) => ExpandoMedia | Promise<ExpandoMedia>;
```

Comment:
In some cases, `detect` tests a string against a RegExp that may return `null` as mentioned in line #19. Given so, `handleLink` must be made aware of that; that `detectResult` may be `null`. I haven't changed it yet cause it breaks all other hosts but added the comment of what would be right.

Reply:
I'm not sure that this would be necessary. The only time `handleLink` is invoked, is after this conditional: https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/46208aab875e262f24b0923e559625453682598d/lib/modules/showImages.js#L856-L857